### PR TITLE
typos

### PR DESCRIPTION
**Pull request by axd1967**
some typos

@alex-w @sushoff

### CODE REVIEW

File: skycultures/greek_leidenAratea/description.en.utf8 [hunk 1]
```diff
@@ -92,7 +92,7 @@ Both, the Almagest and the poem, are written in Greek originally but the Leiden
 	<tr valign="top">
 		<td><notr>Delphinus, Del</notr></td>
 		<td><notr>&Delta;&epsilon;&lambda;&phi;&#943;&sigmaf;</notr></td>
-		<td>Dolphin <br /> In the Greek original (Aratos, ~300 BCE), there was no constellation Equuleus. The Farnese Globe (as well as the Mainz Globe and the Kugel Globe) depicts the Dolphin rahter largely, covering the space of Equ. That is why, I follow this Aratean tradition also for the Leiden Aratea.</td>
+		<td>Dolphin <br /> In the Greek original (Aratos, ~300 BCE), there was no constellation Equuleus. The Farnese Globe (as well as the Mainz Globe and the Kugel Globe) depicts the Dolphin rahter largely, covering the space of Equ. That is why I follow this Aratean tradition also for the Leiden Aratea.</td>
```
Comment (gzotti) [inline skycultures/greek_leidenAratea/description.en.utf8:95]: another typo: rather

**Review by gzotti — COMMENTED** [review]
To me it looks OK. But the description has been authored by @sushoff so she must have the final word!

File: skycultures/greek_leidenAratea/description.en.utf8 (hunk 1 above)
Comment (axd1967) [reply skycultures/greek_leidenAratea/description.en.utf8:95]: That is why I **rather** follow this Aratean tradition **comma** also for ...

Comment (sushoff) [reply skycultures/greek_leidenAratea/description.en.utf8:95]: oh, yes, delete comma

**Comment by alex-w** [comment]
@sushoff is it OK for you?

File: skycultures/greek_leidenAratea/description.en.utf8 (hunk 1 above)
Comment (axd1967) [reply skycultures/greek_leidenAratea/description.en.utf8:95]: @sushoff "rather" was not inserted after all, correct? but I now also see another typo, "rahter".

I think the phrase should read
> ... depicts the Dolphin **rather** largely, covering the space of Equ. That is why I [rather?] follow this Aratean tradition >>,<< also for the Leiden Aratea...

or maybe even better
> ... depicts the Dolphin **rather** largely, covering the space of Equ. That is why I **chose to** follow this Aratean tradition for the Leiden Aratea...

Comment (sushoff) [reply skycultures/greek_leidenAratea/description.en.utf8:95]: yes

**Comment by alex-w** [comment]
Please check the latest stable version of Stellarium:
https://github.com/Stellarium/stellarium/releases/latest

**Pull request merged**